exercises/practice/scrabble-score/.approaches/introduction.md

```diff
@@ -0,0 +1,115 @@
+# Introduction
+
+There are various ways to solve `scrabble-score`.
+This approache document shows different strategies to solve this exercise.
+
+## General guidance
+
+The goal of this exercise is to write a function that calculates the scrabble score for a given word.
+The problem is that the scrabble score is calculated by the sum of the scores of each letter in the word.
+
+## Approach: Using a single dictionary
+
+Using a single dictionary is an approach, it is simple and fast.
+It is also very pythonic.
+
+```python
+LETTER_SCORES = {
+    'A': 1,
+    'E': 1,
+    'I': 1,
+    'O': 1,
+    'U': 1,
+    'L': 1,
+    'N': 1,
+    'R': 1,
+    'S': 1,
+    'T': 1,
+    'D': 2,
+    'G': 2,
+    'B': 3,
+    'C': 3,
+    'M': 3,
+    'P': 3,
+    'F': 4,
+    'H': 4,
+    'V': 4,
+    'W': 4,
+    'Y': 4,
+    'K': 5,
+    'J': 8,
+    'X': 8,
+    'Q': 10,
+    'Z': 10
+}
+
+def score(word):
+    return sum(LETTER_SCORES[letter.upper()] for letter in word)
+```
```

Comment:
```suggestion
```python
LETTER_SCORES = {
    'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1,
    'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
    'D': 2, 'G': 2, 'B': 3, 'C': 3, 'M': 3,
    'P': 3, 'F': 4, 'H': 4, 'V': 4, 'W': 4,
    'Y': 4, 'K': 5, 'J': 8, 'X': 8, 'Q': 10, 'Z': 10
}

def score(word):
    return sum(LETTER_SCORES[letter] for letter in word.upper())
```
```